src/EPR.Payment.Service.UnitTests/Services/Payments/OnlinePaymentsServiceTests.cs

```diff
@@ -0,0 +1,180 @@
+﻿using AutoFixture;
+using AutoFixture.MSTest;
+using AutoMapper;
+using EPR.Payment.Service.Common.Data.Interfaces.Repositories.Payments;
+using EPR.Payment.Service.Common.Data.Profiles;
+using EPR.Payment.Service.Common.Dtos.Request.Payments;
+using EPR.Payment.Service.Common.Dtos.Request.RegistrationFees.ComplianceScheme;
+using EPR.Payment.Service.Common.Dtos.Response.Payments;
+using EPR.Payment.Service.Common.UnitTests.TestHelpers;
+using EPR.Payment.Service.Services.Interfaces.Payments;
+using EPR.Payment.Service.Services.Interfaces.RegistrationFees.ComplianceScheme;
+using EPR.Payment.Service.Services.Payments;
+using EPR.Payment.Service.Services.RegistrationFees.ComplianceScheme;
+using EPR.Payment.Service.Strategies.Interfaces.RegistrationFees.ComplianceScheme;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using FluentValidation;
+using FluentValidation.Results;
+using Moq;
+
+namespace EPR.Payment.Service.UnitTests.Services.Payments
+{
+    [TestClass]
+    public class OnlinePaymentsServiceTests
+    {
+        private Fixture? _fixture = null!;
+        private Mock<IOnlinePaymentsRepository> _onlinePaymentsRepositoryMock = null!;
+        private IMapper _mapper = null!;
```

Comment:
SonarQube: Change type of field '_mapper' from 'AutoMapper.IMapper' to 'AutoMapper.Mapper?' for improved performance

Reply:
done